server_src/lib/logger.js

```diff
@@ -36,44 +45,31 @@ class LoggerService {
         return message;
       }),
     });
-    this.logger = logger;
   }
 
   setLogData(log_data) {
     this.log_data = log_data;
   }
 
-  // async info(message) {
-  //   this.logger.log("info", message);
-  // }
-  async info(message, obj) {
```

Comment:
You know my thoughts. High control over code and its attributes show that we as devs are in control. Not having to await or to then/catch saves a ton of lines of code: 

short, KISS, clean code = bug-free code

I am making a stand in good faith that you will try to pick my brain on how I am so firmly opinionated on this, it is not without reason ;)